package.json

```diff
@@ -8,7 +8,8 @@
     "start": "next start",
     "lint": "next lint",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "prettier": "npx --no prettier . --check"
```

Comment:
Actually, I realize now that we probably _do_ want this to be"prettier:check", because we want a "prettier" script as well to run it without `--check`, probably something like:

`npm run prettier . --`

The -- tells npm to pass any remaining command-line arguments through to prettier.

I'd suggest we just ditch `npx` entirely.